### ISSUE TITLE
Implement a Kafka logging handler

### ISSUE THREAD

**Issue by mlinvill** [SNEWS2/SNEWS_Coincidence_System#104]
Chris and I discussed this topic briefly today. 

We should implement logging over Kafka. This is foundational for centralized monitoring/operations.

**Comment by KaraMelih** [comment]
Would this be on a new topic where only certain admins have access? Is it so that logs can be accessed from different locations?

**Comment by mlinvill** [comment]
Yes, it is configurable. I added a topic argument that lives in the env file, SNEWPLOG_TOPIC. Snews.operations is the default value. At the moment, the implementation uses the same credentials as everything else. I don't think it would be hard to use a separate set of credentials, if it makes sense.

This facilitates log mobility, which would be very useful for an operations center hosted anywhere...or multiple locations. There would be code which reads the topic and writes the log files locally, as needed, where needed.
